pkg/admission/provider.go

```diff
@@ -1,93 +1,91 @@
 // TODO: only supports plain text with no auth, add security later
-// TODO: use keystone to access Neutron API
+// TODO: check keystone expiration date 
 package admission
 
 import (
-	resty "gopkg.in/resty.v1"
+	"fmt"
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/networks"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/ports"
+)
+
+const (
+	providerDefaultRegion  = "RegionOne"
+	providerNeutronService = "neutron"
```

Comment:
other implementations reports other service names, e.g. cloudNetwork, should be
`providerNetworkType = 'network'`
but maybe this option can be removed completly, because NewNetworkV2 cares about this?

Reply:
Right we need to provide only region in the options.